src/services/alerts/alerts.svc.coffee

```diff
@@ -0,0 +1,75 @@
+ angular
+  .module('impac.services.alerts', [])
+  .service('ImpacAlerts', ($log, $q, $http, $httpParamSerializerJQLike, ImpacRoutes, ImpacMainSvc, ImpacEvents, IMPAC_EVENTS)->
+
+    _self = @
+
+    #====================================
+    # Getters
+    #====================================
+    @config = {}
+
+    @config.ssoSessionId = ""
```

Comment:
as discussed / sso_session shouldn't be used there:
- api/v2 will authenticate using basic auth
- js_api/v1 will use Devise